### ISSUE TITLE
Questions about installing Ubuntu onto portable external hard drive with crouton

### ISSUE THREAD

**Issue by ajshell1** [dnschneid/crouton#1439]
I recently bought a 1TB portable external hard drive  (a Westgate Digital My Passport) with the intent of permanently solving the problem of limited storage space on my Toshiba Chromebook 2. However, I have a few questions. 
1. It currently formatted with NTFS. Will I need to format it? I've seen several different answers after looking online. Some people say "No you don't have to", some say "Format it to EXT2", while others say "Format it to EXT4".
2. I have heard that some people have problems with installing directly to external media. Thus, I plan on installing my chroot to the chromebook hard drive and then move it to the external hard drive. What are the proper steps to do this?  I intend to use Unity and trusty in addition to the crouton chrome extension.
3. Once I have the chroot on the external hard drive, what would I type into the console to start the chroot? 
4. Is the any way that I could start the chroot installed on the external hard drive from a windows computer or a chromebook that doesn't have dev mode enabled?
I would like to thank you all for putting up with my unfamiliarity with Linux, and for introducing me to Linux in the first place.

**Comment by DennisLfromGA** [comment]
>  1\. It currently formatted with NTFS. Will I need to format it? I've seen several different answers after looking online. Some people say "No you don't have to", some say "Format it to EXT2", while others say "Format it to EXT4".

- For crouton chroots, make sure the storage is formatted to ext2/3/4.

>  2\. I have heard that some people have problems with installing directly to external media. Thus, I plan on installing my chroot to the chromebook hard drive and then move it to the external hard drive. What are the proper steps to do this? I intend to use Unity and trusty in addition to the crouton chrome extension.

- Once installed, you can move it using `edit-chroot` with the '-m' option -

```
    -m DEST     Moves a chroot. Specify a new name to keep it in the same
                directory, or an absolute path to move it entirely.
                DEST can be a directory, in which case it must end in a slash.
                If multiple chroots are specified, DEST must be a directory.
                If you are moving a chroot to a SD card/USB drive, make sure the
                storage is formatted to ext2/3/4.
```

- So it would be something like:

`sudo edit-chroot precise -m /path/to/external/media/`

>  3\. Once I have the chroot on the external hard drive, what would I type into the console to start the chroot? 

- You can use the full path to the script like: 

`sudo /path/to/external/media/bin/startxfce4`

- Or you can use the '-c' option to specify where your chroots are:

`sudo startxfce4 -c /path/to/external/media/chroots`

>  4\. Is the any way that I could start the chroot installed on the external hard drive from a windows computer or a chromebook that doesn't have dev mode enabled?

- Nope

*Hope this helps,*
-**DennisL**

**P.S.** Sorry for the late reply. :)

**Comment by ajshell1** [comment]
Thanks, but I already figured this stuff out over a year ago.

Since then, I've bought a gaming desktop and a normal laptop, and I've given my Chromebook to my Grandmother, who can actually use it without feeling constrained.

I've gained a lot of experience with Linux thanks to Crouton, but I'm glad I use a computer that has upgradable internal storage instead of a Chromebook. Maybe I'll eventually use Linux instead of Windows someday. Maybe.

Also, I just noticed that I referred to "Western Digital" as "Westgate Digital".
